client/we_meet_client/src/__features__/rsvp_to_event.feature.js

```diff
@@ -21,7 +21,7 @@ describe('visitor can RSVP to event', () => {
 
   describe('when user logged in', () => {
 
-    it('displays updated attendee list upon rsvp', async () => {
+    xit('displays updated attendee list upon rsvp', async () => {
 
```

Comment:
There's 2 'it' blocks 'Xed' out here.  Does this need to be reverted?

Reply:
Nope. This tests are not going to work. That functionality is removed at the moment.